Tools/ChangeLog

```diff
@@ -1,3 +1,22 @@
+2022-03-14  Jonathan Bedard  <jbedard@apple.com>
+
+        [Merge-Queue] Fail draft PRs
```

Comment:
Fail => Skip

Reply:
The choice of "Fail" here was deliberate, see the discussion about "Should the build be `FAILED`, or `SKIPPED`?"